Library/Koala/Koala.swift

```diff
@@ -1824,6 +1824,30 @@ public final class Koala {
     }
   }
 
+  public func trackViewedPaymentMethods() {
+     self.track(event: "Viewed Payment Methods")
+  }
+
+  public func trackViewedAddNewCard() {
+    self.track(event: "Viewed Add New Card")
+  }
+
+  public func trackDeletedPaymentMethod() {
+    self.track(event: "Deleted Payment Method")
+  }
+
+  public func trackErroredDeletePaymentMethod() {
```

Comment:
Looks like the only other function in `KoalaSwift` that uses `trackErrorred....` format is `trackStripeTokenErroredForApplePay` everything else prefers `trackDeletePaymentMethodError` format. Should we unify this and rename this to `trackDeletePaymentMethodError`?